internal/mdxc/application_identity.go

```diff
@@ -28,9 +28,9 @@ type AWSApplicationIdentityOutputData struct {
 	IAMRoleARN types.String `tfsdk:"iam_role_arn"`
 }
 type AzureApplicationIdentityOutputData struct {
-	ApplicationID          types.String `tfsdk:"application_id"`
```

Comment:
Why are we removing `ApplicationId`? Is it not useful?  I believe the only way to look up the Application (for `Read` or `Update`) is via the application ID.  Here is the query: https://github.com/massdriver-cloud/terraform-provider-mdxc/pull/28/files?diff=split&w=1#diff-1c8618fd6e267412ff00f04541e24eaca469da6aea403c53bd94d295201aa585R71

If we need to **add** ServicePrincipalClientID thats fine, but we shouldn't remove ApplicationID

Reply:
I agree `ApplicationID` is still useful. Newer commits brought it back, but I missed this line. Next commit will revert this change here.

Reply:
We need it in a few places:
https://github.com/massdriver-cloud/terraform-provider-mdxc/pull/28/files#diff-08d112e976e2479674666bd6acea8430fd217225c39ce5186b3e1a4788d52ad1L82
https://github.com/massdriver-cloud/terraform-provider-mdxc/pull/28/files#diff-08d112e976e2479674666bd6acea8430fd217225c39ce5186b3e1a4788d52ad1L48
https://github.com/massdriver-cloud/terraform-provider-mdxc/pull/28/files#diff-19a6e506204b4de650e1289aa75ded86e795860b5da91100431d3c685fc8fe1dL160